eng/config/PublishData.json

```diff
@@ -72,12 +72,12 @@
       "vsBranch": "rel/d16.6",
       "vsMajorVersion": 16
     },
-    "release/dev16.7-preview2-vs-deps": {
+    "release/dev16.7-preview3-vs-deps": {
       "nugetKind": ["Shipping", "NonShipping"],
       "version": "3.7.*",
       "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
       "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
-      "channels": [ "dev16.7", "dev16.7p2" ],
+      "channels": [ "dev16.7", "dev16.7p3" ],
```

Comment:
I think a channel should only be in one branch, right @jaredpar? Otherwise this entry and the master-vs-deps entry might be fighting?

Reply:
Correct.

Reply:
What specifically is the undesired effect of this?